scripts/helpers/saveAndRestoreFile.js

```diff
@@ -7,7 +7,9 @@ module.exports = ({ file, extension = ".tmp" }) => {
     fs.renameSync(file, backupFile);
 
     const cleanup = () => {
-      fs.renameSync(backupFile, file);
+      if (fs.existsSync(backupFile)) {
```

Comment:
When I try to stop the functional tests using CTRL+C on my machine, I saw an error where it said that the backupFile did not exist anymore.